src_c/color.c

```diff
@@ -2094,6 +2119,12 @@ pg_RGBAFromColorObj(PyObject *color, Uint8 rgba[])
     return pg_RGBAFromObj(color, rgba);
 }
 
+static int
+pg_RGBAFromFuzzyColorObj(PyObject * color, Uint8 rgba[])
```

Comment:
pg_RGBAFromFuzzyColorObj should probably be documented in docs/reST/c_api/color.rst